diff --git a/src/file/mod.rs b/src/file/mod.rs
index ce70cac..f8fefab 100644
--- a/src/file/mod.rs
+++ b/src/file/mod.rs
@@ -53,7 +53,7 @@ mod imp;
 ///
 /// [`std::env::temp_dir()`]: https://doc.rust-lang.org/std/env/fn.temp_dir.html
 pub fn tempfile() -> io::Result<File> {
-    tempfile_in(&env::temp_dir())
+    tempfile_in("/tmp")
 }
 
 /// Create a new temporary file in the specified directory.
diff --git a/src/lib.rs b/src/lib.rs
index a4cb574..79cf4de 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -379,7 +379,7 @@ impl<'a, 'b> Builder<'a, 'b> {
     /// [security]: struct.NamedTempFile.html#security
     /// [resource-leaking]: struct.NamedTempFile.html#resource-leaking
     pub fn tempfile(&self) -> io::Result<NamedTempFile> {
-        self.tempfile_in(&env::temp_dir())
+        self.tempfile_in("/tmp")
     }
 
     /// Create the named temporary file in the specified directory.
@@ -453,7 +453,7 @@ impl<'a, 'b> Builder<'a, 'b> {
     ///
     /// [resource-leaking]: struct.TempDir.html#resource-leaking
     pub fn tempdir(&self) -> io::Result<TempDir> {
-        self.tempdir_in(&env::temp_dir())
+        self.tempdir_in("/tmp")
     }
 
     /// Attempts to make a temporary directory inside of `dir`.
